File: invesdwin-context-parent/invesdwin-context-scala/src/test/java/de/invesdwin/context/scala/tests/InputsAndResultsTestBoolean.scala
System.out.println("getBoolean")
if(getBoolean != null)
	throw new Exception("getBoolean already defined!")
val getBoolean = putBoolean
val getBooleanType = getBoolean.getClass()
System.out.println(getBooleanType)
System.out.println(getBoolean)
if(getBooleanType != classOf[java.lang.Boolean])
	throw new Exception("getBoolean not Boolean!")

System.out.println("getBooleanVector")
if(getBooleanVector != null)
	throw new Exception("getBooleanVector already defined!")
val getBooleanVector = putBooleanVector.asInstanceOf[Array[Boolean]]
val getBooleanVectorType = getBooleanVector(0).getClass()
System.out.println(getBooleanVectorType)
System.out.println(getBooleanVector)
if(getBooleanVectorType != classOf[Boolean])
	throw new Exception("getBooleanVector not Boolean!")

System.out.println("getBooleanVectorAsList")
if(getBooleanVectorAsList != null)
	throw new Exception("getBooleanVectorAsList already defined!")
val getBooleanVectorAsList = putBooleanVectorAsList.asInstanceOf[Array[Boolean]]
val getBooleanVectorAsListType = getBooleanVectorAsList(0).getClass()
System.out.println(getBooleanVectorAsListType)
System.out.println(getBooleanVectorAsList)
if(getBooleanVectorAsListType != classOf[Boolean])
	throw new Exception("getBooleanVectorAsList not Boolean!")

System.out.println("getBooleanMatrix")
if(getBooleanMatrix != null)
	throw new Exception("getBooleanMatrix already defined!")
val getBooleanMatrix = putBooleanMatrix.asInstanceOf[Array[Array[Boolean]]]
val getBooleanMatrixType = getBooleanMatrix(0)(0).getClass()
System.out.println(getBooleanMatrixType)
System.out.println(getBooleanMatrix)
if(getBooleanMatrixType != classOf[Boolean])
	throw new Exception("getBooleanMatrix not Boolean!")

System.out.println("getBooleanMatrixAsList")
if(getBooleanMatrixAsList != null)
	throw new Exception("getBooleanMatrixAsList already defined!")
val getBooleanMatrixAsList = putBooleanMatrixAsList.asInstanceOf[Array[Array[Boolean]]]
val getBooleanMatrixAsListType = getBooleanMatrixAsList(0)(0).getClass()
System.out.println(getBooleanMatrixAsListType)
System.out.println(getBooleanMatrixAsList)
if(getBooleanMatrixAsListType != classOf[Boolean])
	throw new Exception("getBooleanMatrixAsList not Boolean!")

File: invesdwin-context-parent/invesdwin-context-jruby/src/test/java/de/invesdwin/context/jruby/tests/InputsAndResultsTestBoolean.rb
print("getBoolean")
if (defined?(getBoolean)).nil?
	raise "getBoolean already defined!"
getBoolean = putBoolean
getBooleanType = getBoolean.class
print(getBooleanType)
print(getBoolean)
if(getBooleanType != 'boolean')
	raise "getBoolean not boolean!"

print("getBooleanVector")
if (defined?(getBooleanVector)).nil?
	raise "getBooleanVector already defined!"
getBooleanVector = putBooleanVector
getBooleanVectorType = getBooleanVector[0].class
print(getBooleanVectorType)
print(getBooleanVector)
if(getBooleanVectorType != 'boolean')
	raise "getBooleanVector not boolean!"

print("getBooleanVectorAsList")
if (defined?(getBooleanVectorAsList)).nil?
	raise "getBooleanVectorAsList already defined!"
getBooleanVectorAsList = putBooleanVectorAsList
getBooleanVectorAsListType = getBooleanVectorAsList[0].class
print(getBooleanVectorAsListType)
print(getBooleanVectorAsList)
if(getBooleanVectorAsListType != 'boolean')
	raise "getBooleanVectorAsList not boolean!"

print("getBooleanMatrix")
if (defined?(getBooleanMatrix)).nil?
	raise "getBooleanMatrix already defined!"
getBooleanMatrix = putBooleanMatrix
getBooleanMatrixType = getBooleanMatrix[0][0].class
print(getBooleanMatrixType)
print(getBooleanMatrix)
if(getBooleanMatrixType != 'boolean')
	raise "getBooleanMatrix not boolean!"

print("getBooleanMatrixAsList")
if (defined?(getBooleanMatrixAsList)).nil?
	raise "getBooleanMatrixAsList already defined!"
getBooleanMatrixAsList = putBooleanMatrixAsList
getBooleanMatrixAsListType = getBooleanMatrixAsList[0][0].class
print(getBooleanMatrixAsListType)
print(getBooleanMatrixAsList)
if(getBooleanMatrixAsListType != 'boolean')
	raise "getBooleanMatrixAsList not boolean!"
